packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -87,7 +87,7 @@ This is a starting point for browser-based OpenID Connect flows such as the impl
 | display                          | The `display` parameter to be passed to the Social Identity Provider when performing Social Login.                                                                                                                                                                                                                                                                                                                                                 | Query       | String    | FALSE      |
 | idp_scope                        | A space delimited list of scopes to be provided to the Social Identity Provider when performing [Social Login](/docs/concepts/social-login/). These scopes are used in addition to the scopes already configured on the Identity Provider.                                                                                                                                                                                                  | Query       | String    | FALSE      |
 | [idp](/docs/reference/api/idps/) | Identity provider to use if there is no Okta Session.                                                                                                                                                                                                                                                                                                                   | Query       | String    | FALSE      |
-| login_hint                       | A username to prepopulate if prompting for authentication.                                                                                                                                                                                                                                                                                                                                                                                         | Query       | String    | FALSE      |
+| login_hint                       | A username to prepopulate if prompting for authentication. NOTE: This parameter will be ignored if the idp parameter is used.                                                                                                                                                                                                                                                                                                                                                                                         | Query       | String    | FALSE      |
```

Comment:
imo a note like this belongs on the `idp` param docs, not the `login_hint` docs

Reply:
^I think this point can be mentioned on both, since it involves both of them.

On the `idp` param, we could note that specifying `idp` will cause the `login_hint` to be ignored, and on the `login_hint` param, we could say something like: `A username to prepopulate if prompting for authentication and an idp is not specified`.